alf/bin/train_test.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tempfile
+import os
+import io
+import subprocess
+from pathlib import Path
+import unittest
+import numpy as np
+
+from absl import logging
+from absl.testing import parameterized
+
+from tensorboard.backend.event_processing import event_file_loader
+from tensorboard.util import tensor_util
```

Comment:
tb_util might be a better name

Reply:
tensor_util is used for deserialize a tensor from bytes , the name is ok